warehouse/static/sass/settings/_colours.scss

```diff
@@ -14,14 +14,14 @@
 
 // SETTINGS - COLOURS
 
-$brand-color:                 #006dad;
-$primary-color:               #ffd343;
+$primary-color:                 #006dad;
+$highlight-color:               #ffd343;
```

Comment:
There still appears to be an indentation issue here.  All variables should line up. Check that you're not using tabs rather than spaces :)